src/Microsoft.Owin.Security.ActiveDirectory/IssuerSigningKeys.cs

```diff
@@ -19,6 +19,6 @@ internal class IssuerSigningKeys
         /// <summary>
         /// Signing tokens.
         /// </summary>
-        public IEnumerable<X509SecurityToken> Tokens { get; set; }
+        public IEnumerable<SecurityKey> Keys { get; set; }
```

Comment:
You should take that up with @lovemaths and @brentschmaltz, most of this was done on their advice.